torch/_refs/__init__.py

```diff
@@ -5087,6 +5087,17 @@ def bucketize(
     return start.to(dtype=out_dtype)
 
 
+@register_decomposition(aten.log_normal)
+@out_wrapper()
+@elementwise_type_promotion_wrapper(
+    type_promoting_args=("self",),
+    type_promotion_kind=ELEMENTWISE_TYPE_PROMOTION_KIND.DEFAULT,
+)
+def log_normal(self, mean=1, std=2, generator=None):
+    assert generator is None
+    return torch.exp(std * torch.randn_like(self) + mean)
```

Comment:
I wonder if special care is needed to handle overflow, underflow for `log_normal`. For fp32, `torch.exp(x)` overflows for `x >= 89`. 

`torch.randn` practically won't cause this overflow as 99% of values will be in [-3, 3].
But for some `mean`, `std`, this will cause overflow.  
```
>>> torch.empty(100000).log_normal_(mean=89).max()
tensor(inf)
```
I wonder if this behavior is expected in `log_normal_` eager mode.

```
>>> np.random.lognormal(89, 1, 100000).max()
2.9258184949413217e+40
```

Reply:
I think this is alright, but thank you for bringing it up!